java/ql/lib/semmle/code/java/frameworks/spring/SpringBoot.qll

```diff
@@ -0,0 +1,24 @@
+/**
+ * Provides classes for working with Spring classes and interfaces from
+ * `org.springframework.boot.*`.
+ */
+
+import java
+
+/**
+ * The class `org.springframework.boot.actuate.autoconfigure.security.servlet.EndpointRequest`.
+ */
+class TypeEndpointRequest extends Class {
```

Comment:
Most of the classes under `semmle.code.java.frameworks.spring` are prefixed with the name `Spring`. Do you think this would make sense for these classes as well?

```suggestion
class SpringTypeEndpointRequest extends Class {
```